Shared/Bytes.swift

```diff
@@ -11,8 +11,13 @@ public typealias GUID = String
  */
 open class Bytes {
     open class func generateRandomBytes(_ len: UInt) -> Data {
-        let bytes = [UInt32](repeating: 0, count: Int(len)).map { _ in arc4random() }
-        return Data(bytes: bytes, count: Int(len))
+        let len = Int(len)
+        var data = Data(count: len)
+        data.withUnsafeMutableBytes { (p: UnsafeMutablePointer<UInt8>) in
+            let result = SecRandomCopyBytes(kSecRandomDefault, len, p)
+            assert(result == 0, "Random byte generation failed.")
```

Comment:
This moves the computation outside of the assertion as was recommended here: https://bugs.swift.org/browse/SR-4308